src/index.html

```diff
@@ -15,8 +15,92 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO LOGO"
+          class="header__logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__ul">
+          <li class="header__li">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="header__li">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="header__li">
```